### PR TITLE
Document the requirement to setup remote maven repository

### PR DESCRIPTION
**Pull request by sabbyanandan**
Resolves spring-cloud/spring-cloud-dataflow-server-cloudfoundry#139

### CODE REVIEW

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -78,12 +78,22 @@ cf set-env dataflow-server SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_PASSWORD {password
 cf set-env dataflow-server SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SKIP_SSL_VALIDATION false
 ```
 
+Spring Cloud Data Flow server implementations (local, cf, mesos, yarn, or kubernetes) do not have
```
Comment (markfisher) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: not true for *local*

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -78,12 +78,22 @@ cf set-env dataflow-server SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_PASSWORD {password
...
+'any' default remote maven repository configured. This is intentionally designed to provide the flexibility for
+the users, so they can override and point to a remote repository of their choice. The out-of-the-box
+applications that are supported by Spring Cloud Data Flow are available in Spring's repository,
+so if you want to use them, you 'must' set it as the remote repository as listed below.
+
+```
+cf set-env dataflow-server MAVEN_REMOTE_REPOSITORIES_REPO1_URL https://repo.spring.io/libs-snapshot
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: We can specify that `REPO1` is the `alias` name for the repository.

**Comment by ilayaperumalg** [comment]
LGTM, merging after fixing the above comment.

**Comment by ilayaperumalg** [comment]
Squashed and merged as 99fdc13

**Comment by sabbyanandan** [comment]
Thanks, @ilayaperumalg.

**Pull request closed without merging**